doc/PROTOCOL-HTTP2.md

```diff
@@ -136,6 +137,10 @@ implementation can decode valid portions while leaving broken %-encodings as-is
 or replacing them with a replacement character (e.g., '?' or the Unicode
 replacement character).
 
+Servers can optionally send **Status-Details** and by default the value is a
+serialized protobuf message of type
+[google.rpc.Status](https://github.com/googleapis/googleapis/blob/master/google/rpc/status.proto).
```

Comment:
I'm not sure how to interpret this - what should users expect when the content-type is `application/grpc+json` or `application/grpc+tcompact`?